src/main/steps/applicant/upload-document/upload-documents-list.ts

```diff
@@ -0,0 +1,171 @@
+import * as URL from '../../urls';
+
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+
+export const generateUploadDocumentList = (sectionTitles, taskListItems) => {
+  return [
+    {
+      title: sectionTitles.witness_statements_and_evidence,
+      items: [
+        {
+          id: 'your-position-statements',
+          text: taskListItems.your_position_statements,
+          href:
```

Comment:
Looks repetitive logic for href value, see if we can create a common method. This will reduce number of lines of code